### PR TITLE
[UXTHEME] Localize string resources

### PR DESCRIPTION
**Pull request by binarymaster**
JIRA issue: [CORE-5991](https://jira.reactos.org/browse/CORE-5991) - addendum to 118869f69 / #5813.

- Adapt SetWindowResourceText() function from
  `msconfig_new/utils.c` LoadResourceStringEx() function.
- Drop `version.rc` in favor of `uxtheme.rc` since we have forked uxtheme.
- Localize string resources.

Cc @Splitwirez for review.

### CODE REVIEW

File: dll/win32/uxtheme/uxtheme.rc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#pragma code_page(65001)
+
+#ifdef LANGUAGE_EN_US
+    #include "lang/en-US.rc"
+#endif
+#ifdef LANGUAGE_RU_RU
+    #include "lang/ru-RU.rc"
+#endif
+
+#include "version.rc"
```
Comment (HBelusca) [inline dll/win32/uxtheme/uxtheme.rc:23]: version.rc ?

**Comment by Splitwirez** [comment]
So uh...I am so sorry for not getting back to you on this PR sooner - I got sidetracked before getting to properly look at this yesterday.

Regrettably, I still can't properly look at it right now, but I did have a moment, so I'd like to quickly drop these here for you before I have the chance to forget:

- Is the text in the Classic Theme preview already localized?

- If so...should/does this PR use that same text for the Visual Style preview?

**Comment by binarymaster** [comment]
> Is the text in the Classic Theme preview already localized?

Yes.

> If so...should/does this PR use that same text for the Visual Style preview?

No, I've verified with Windows XP, these strings are stored within uxtheme.dll.

File: dll/win32/uxtheme/CMakeLists.txt [hunk 2]
```diff
@@ -12,6 +12,7 @@ list(APPEND SOURCE
     ncscrollbar.c
     nonclient.c
     property.c
+    resource.h
```
Comment (katahiromz) [inline dll/win32/uxtheme/CMakeLists.txt:15]: Why compile `resource.h`? I don't think precompiled header is useful in this situation.

Comment (binarymaster) [reply dll/win32/uxtheme/CMakeLists.txt:15]: `uxthemep.h` is also included here, isn't this also for dependency tracking?

File: dll/win32/uxtheme/nonclient.c [hunk 3]
```diff
@@ -1218,7 +1241,7 @@ HRESULT WINAPI DrawNCPreview(HDC hDC,
     if (!RegisterClassExW(&DummyPreviewWindowClass))
         return E_FAIL;
 
-    hwndDummy = CreateWindowExW(0, L"DummyPreviewWindowClass", L"Active window", WS_OVERLAPPEDWINDOW | WS_VSCROLL, 30, 30, 300, 150, 0, 0, hDllInst, NULL);
+    hwndDummy = CreateWindowExW(0, L"DummyPreviewWindowClass", L"", WS_OVERLAPPEDWINDOW | WS_VSCROLL, 30, 30, 300, 150, 0, 0, hDllInst, NULL);
```
Comment (katahiromz) [inline dll/win32/uxtheme/nonclient.c:1244]:
```suggestion
    hwndDummy = CreateWindowExW(0, L"DummyPreviewWindowClass", NULL,
                                WS_OVERLAPPEDWINDOW | WS_VSCROLL,
                                30, 30, 300, 150, NULL, NULL, hDllInst, NULL);
```

File: dll/win32/uxtheme/CMakeLists.txt (hunk 2 above)
Comment (binarymaster) [reply dll/win32/uxtheme/CMakeLists.txt:15]: Yes, it's for dependency tracking, see other modules:
- https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/mstsc/CMakeLists.txt;hb=e0b67a29901cfe1ef11608fd72a24c48aa9525f1
- https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/rapps/CMakeLists.txt;hb=e0b67a29901cfe1ef11608fd72a24c48aa9525f1

File: dll/win32/uxtheme/nonclient.c [hunk 4]
```diff
@@ -1266,10 +1299,17 @@ HRESULT WINAPI DrawNCPreview(HDC hDC,
...
 
     SetTextColor(hDC, GetThemeSysColor(context.theme, TMT_WINDOWTEXT));
-    DrawThemeText(context.theme, hDC, WP_DIALOG, 0, L"Window Text", -1, DT_LEFT | DT_TOP | textDrawFlags, 0, &rcWindowClient);
+    len = LoadStringW(hDllInst, IDS_WINTEXT, (LPWSTR)&szText, 0);
+    if (len > 0)
+        DrawThemeText(context.theme, hDC, WP_DIALOG, 0, szText, len, DT_LEFT | DT_TOP | textDrawFlags, 0, &rcWindowClient);
 
     /* Draw preview dialog window */
-    SetWindowTextW(hwndDummy, L"Message Box");
+    szText = LoadResourceString(hDllInst, IDS_MESSAGEBOX);
```
Comment (HBelusca) [inline dll/win32/uxtheme/nonclient.c:1307]: hmm, why are you sometimes using LoadStringW, and some other times this LoadResourceString function?

Comment (binarymaster) [reply dll/win32/uxtheme/nonclient.c]: Thinking about your question, I've simplified the code a bit now. 🙂

**Review by HBelusca — APPROVED** [review]
lgtm!

**Pull request merged**